scrapy/utils/ssl.py

```diff
@@ -1,14 +1,9 @@
-import OpenSSL
+import OpenSSL.SSL
 import OpenSSL._util as pyOpenSSLutil
 
 from scrapy.utils.python import to_unicode
 
 
-# The OpenSSL symbol is present since 1.1.1 but it's not currently supported in any version of pyOpenSSL.
-# Using the binding directly, as this code does, requires cryptography 2.4.
-SSL_OP_NO_TLSv1_3 = getattr(pyOpenSSLutil.lib, 'SSL_OP_NO_TLSv1_3', 0)
```

Comment:
We can't deprecate this without some magic (probably the least magic way is the module-level `__getattr__`).

If we decide to deprecate it, this line can be changed into `SSL_OP_NO_TLSv1_3 = OpenSSL.SSL.OP_NO_TLSv1_3` and the deprecation will be done separately.

Reply:
No strong opinion either way from me.